Content.Client/UserInterface/Systems/Actions/ActionUIController.cs

```diff
@@ -627,22 +627,20 @@ private void OnFilterSelected(ItemPressedEventArgs args)
         QueueWindowUpdate();
     }
 
-    private void OnWindowActionPressed(GUIBoundKeyEventArgs args, ActionButton action)
+    private void OnWindowActionPressed(GUIBoundKeyEventArgs args, ActionButton button)
     {
         if (args.Function != EngineKeyFunctions.UIClick && args.Function != EngineKeyFunctions.Use)
             return;
 
-        _menuDragHelper.MouseDown(action);
-        args.Handle();
```

Comment:
```suggestion
        HandleActionPressed(args, button); // Moffstation
```

Our contributions guidelines require single-line changes to existing files be denoted with trailing `// Moffstation` comments to make it more clear what's going on when merge conflicts come up.